src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.scss

```diff
@@ -0,0 +1,18 @@
+:host,
```

Comment:
is this 1st `host` needed?

Reply:
Yes, is just for grouping properties and don't repeat them.